citest/google_session/__init__.py

```diff
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Citest support modules for automating Google sessions and authenticated requests."""
+"""Citest support modules for automating Google sessions and authenticated
+requests.
```

Comment:
This is neither a Spinnaker or GCP module -- it is really a Google authentication module. You could use this to automate signing in for any Google service (e.g. Mail, Google+, etc).